readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,96 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: []
+formats: any(list(enum('htmlzip', 'pdf', 'epub')), enum('all'), required=False)
```

Comment:
This is only valid for sphinx, I wonder if we can move this to the sphinx key, or maybe we want to support this https://github.com/rtfd/readthedocs.org/issues/1939? (still no pdf for mkdocs)

Reply:
It's only valid for sphinx *for now*. We might have other engines that produce a subset of these later. I'm fine with this being top level